src/errors.rs

```diff
@@ -45,6 +45,8 @@ pub enum Error {
     InvalidRamBundleEntry,
     /// Tried to operate on a non RAM bundle file
     NotARamBundle,
+    /// Range mapping index is invalid
+    InvalidRangeMappingIndex(data_encoding::DecodeError),
```

Comment:
Just a NOTE: this is a breaking change as `Error` is not (yet) marked as `#[non_exhaustive]`

Reply:
I would say if we're ok with making a breaking change here, then we should also put the information whether a token is a range token directly on `RawToken`.